x-pack/plugins/maps/public/classes/layers/file_upload_wizard/config.tsx

```diff
@@ -9,12 +9,21 @@ import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { LayerWizard, RenderWizardArguments } from '../../layers/layer_wizard_registry';
 import { ClientFileCreateSourceEditor, INDEX_SETUP_STEP_ID, INDEXING_STEP_ID } from './wizard';
+import { getFileUpload } from '../../../kibana_services';
 
 export const uploadLayerWizardConfig: LayerWizard = {
   categories: [],
   description: i18n.translate('xpack.maps.fileUploadWizard.description', {
     defaultMessage: 'Index GeoJSON data in Elasticsearch',
   }),
+  disabledReason: i18n.translate('xpack.maps.fileUploadWizard.disabledDesc', {
+    defaultMessage:
+      'You do not have file upload privileges. Ask your system administrator to grant file upload privileges.',
```

Comment:
nit: this message isn't very actionable. We don't have anything within the role management screen called "file upload privileges". Can we explain which privileges are required here?

Reply:
I wanted to avoid putting too many details in the message and instead include permissions in the documentation. I can include the privileges in the UI but figured it would best to document them in a single location to avoid having multiple messages around kibana UI.